rest/src/main/resources/META-INF/beans.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<beans
+    xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_2_0.xsd"
+    bean-discovery-mode="all">
+</beans>
```

Comment:
Please use CDI 3.0 one.

Reply:
Done

Reply:
Looks like you only adjusted the one in `/cdi`. Please also adjust abovementioned one :)

Reply:
Done